api/basic/schema.ts

```diff
@@ -0,0 +1,50 @@
+export interface Response {
+  statusCode: number
+  body: any
+}
+
+export interface Pin {
+  cid: string
+  status: 'queued' | 'pinning' | 'pinned' | 'failed' | 'unpinned'
+  created: string
+}
+
+export interface ClusterStatusResponse {
+  'cid': string
+  'name': ''
+  'allocations': []
+  'origins': []
+  'created': string
+  'metadata': null
+  'peer_map': {
+    '12D3KooWArSKMUUeLk3z2m5LKyb9wGyFL1BtWCT7Gq7Apoo77PUR': {
+      'peername': 'elastic-ipfs'
+      'ipfs_peer_id': string
+      'ipfs_peer_addresses': string[]
+      'status': Pin['status']
+      'timestamp': string
+      'error': ''
+      'attempt_count': 0
+      'priority_pin': false
+    }
+  }
+}
+
+export interface ClusterAddResponse {
+  replication_factor_min: -1
+  replication_factor_max: -1
+  name: ''
+  mode: 'recursive'
+  shard_size: 0
+  user_allocations: null
+  expire_at: '0001-01-01T00:00:00Z'
```

Comment:
this shouldn't be hardcoded timestamp?

Reply:
agree these look weird, but they currently capture the "fakeness" of our cluster responses. We don't support expiring pins here, so the value will always be that one. I can change the type defs if you feel strongly about it tho

Reply:
don't feel, so maybe just add a comment :)